lib/bpf/bpf_test.go

```diff
@@ -165,25 +162,25 @@ func (s *Suite) TestObfuscate(c *check.C) {
 	go func() {
 		// Create temporary file.
 		file, err := os.CreateTemp("", "test-script")
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
 		defer os.Remove(file.Name())
 
 		// Write script to file.
 		shellContents := fmt.Sprintf("#!%v\necho bHM= | %v --decode | %v",
 			shellPath, decoderPath, shellPath)
 		_, err = file.Write([]byte(shellContents))
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
 		err = file.Close()
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
 
 		// Make script executable.
 		err = os.Chmod(file.Name(), 0700)
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
```

Comment:
I think we could use `t.TempDir()` to create the directory and `os.WriteFile()` to create a file and set correct permissions.